composer.json

```diff
@@ -10,7 +10,7 @@
 		}
 	],
 	"require": {
-		"silverstripe/comments": ">=1.2"
+		"silverstripe/comments": "^2.0"
```

Comment:
Problems:

* `>=1.2` was too loose and was pulling in comments SS4 compatible (3.x-dev)
* `^1.2` builds don't pass as the API changed and this module's tests seem to be expecting 2.0 or higher

`>=1.2` would historically have been pulling in 2.0 or higher anyway, it's just that now we have an SS4 compatible 3.x which shows that this is flawed.

The 2.0 release is two years old now, so locking it to this.